packages/common/src/types/dbt.ts

```diff
@@ -110,6 +111,8 @@ export const normaliseModelDatabase = (
             return { ...model, database: model.database as string };
         case SupportedDbtAdapter.DATABRICKS:
             return { ...model, database: model.database || 'DEFAULT' };
+        case SupportedDbtAdapter.TRINO:
+            return { ...model, database: model.database || 'DEFAULT' };
```

Comment:
I can see you copied the databricks case block but that is an edge case. We can move this case to line 104 and behave like the other warehouses.